src/Orchard.Web/Modules/Orchard.Widgets/Drivers/WidgetPartDriver.cs

```diff
@@ -32,6 +34,14 @@ public class WidgetPartDriver : ContentPartDriver<WidgetPart> {
             widgetPart.AvailableZones = _widgetsService.GetZones();
             widgetPart.AvailableLayers = _widgetsService.GetLayers();
 
+            if (widgetPart.LayerPart == null) {

```

Comment:
Please test on the `ILocalizableAspect` instead. 

    var localizableAspect = widgetPart.As<ILocalizableAspect>();
    if(localizationAspect != null ) {
        ...
    }